fastapi_cache/decorator.py

```diff
@@ -125,15 +125,15 @@ async def ensure_async_func(*args: P.args, **kwargs: P.kwargs) -> R:
                 )
             try:
                 ttl, ret = await backend.get_with_ttl(cache_key)
-            except ConnectionError:
+            except Exception:
```

Comment:
I think it's still a better default than failing but I agree we probably shouldn't just swallow this - I've added some default logging. Happy to merge?
As per #110 it would be good to do a new release too as PickleCoder is broken in the PyPI version right now. I've closed #110 as all the changes are in this PR now.

Reply:
Hi @long2ice you happy with the new version? Could we get this merged and a new release on PyPI?

Reply:
@mkdir700 any chance you can help with this?